protos/profiles.proto

```diff
@@ -30,3 +30,13 @@ message BigQuery {
 
   reserved 2;
 }
+
+message SQLDataWarehouse {
+  string server = 2;
+  int32 port = 3;
+  string username = 4;
+  string password = 5;
+  string database_name = 6;
+
+  reserved 1;
```

Comment:
don't bother with this, you can just renumber all of your fields since this protobuf has never been used / changed before.